styles/main.header.scss

```diff
@@ -33,3 +33,114 @@
x5T3BY9AydBzxHK/Rteomu0g/pBv2Sfk236Q/0F/ob4LxAiC5QDseX1SrWSqkVuqL2+KoqybFMEYrROu6UcKeMaylly527NIg7Ofy9Rifl7gHdVFmscgFXbtIqKOFV11SF3sb8E+YQTeBKEvMy5j3M+5gPBPcFvHsJMyK4LoCKi2oXmE5QHkcLuCYoSSk6SaeA9Sw9C8ov0WV6Gbhu01v0e7pH9+kBhYe/xDIYXBj8OZ5+CpKsKltVVJUstU9HMXJqRZVUFBSug7Z1SKWnoe6oR3hXj6icZdW+KuJOCVT2ONROy5U9voq1GPYAXekDdlkFfO2o5frzTFMPI6vpAdasulUrqnW8b0Hqbwlnu6oM2Gce1V3MamdeQUUOQKDHVtvO01jBNvcrOO8qL9jzkWjckazaDrKmKsD6885bdU2B4u0nJwYSKQXhi/E12kmvlurDVztM0RYa+5I8PMeTD/aSlirAi2C92r5DibHxbWNHIg9jVduOlsUuy/3sQ/+92PJE6Ym1/6E/PuwH8Snq310WCMOPjgKG4VFjNAIfmaI4vOpJXMtJLHgWAH8MGKSLgCH42UuIeVcASboOmKAtwBg8700ah/e9DY/7ASCBtyfgh4fw9BBwnaRZ+G+iGcxFmoOHWMCag4aS45jDEgn10L+jLqVTZmZMzONYFwGtzhgyETUhMI1ZH5OAhJkWHQNVaY6uOMa6yCfZ5rpj4Sn3yrAHRlwuGseUC+R5IiLHgQZIuOAMTX/9agJcaEgLJ1O4EulqMFHAGLQ0ZI5abkkPH9OAENYahWyc+MrR9SK0fgXa3pII+wGFR99heQ9/ZyQOyr7ga8157LttvJ8/Er9LyAdybqzUY0VlaxXEFmBRt2gVu7Wo4GvNvt2XKFOVOFYAVE0EtLCjc+32J/jI9bNPgaU1+lZ9Ipkbl90MgqVkHZmUHAwlT+wutPPWzblCIAxFJ3I5HANXAflChaMz/vqP8apl4a7q+tOS65NZA0Kph8Zyq/XU420Q+tuPWtGl13Jo8OpGLcu0BHK1PCxnTXILpvoWzsvC44on+u+4lJU99OU8cmzAULMlnsn6wLDOGOpy4XxIzviJp508jGXQEm+bbBM2n+0WkRtjsVnDZ1ciOi5rnKLFUjsrDmonrqy9O8nyfTLvb8P9xE1/6vqL4f3F9V6kZHZJE3/BqA2sh09hsJc9Qh4sH1vaDYYheEb/qUipOR+bl/gfl8geklg/iPicxDlnMeMiiSTifRoRnDOJFDKiDN7juD2B3OYYIvwMYBTZ1ixi+HFAAnnXCUT6y4AQXQPEaRMQoVcBg/Q6YID+Tv/AeRkwQu8BhpG73cPz/wSM0AFgGDX1fVx5AAjR+4AIVQAD+CuG+xGR5WnMAfoMMrxTqLhnQcsJ+iyoO07nAHrMSy5yBpSdEdC/SfOX/ptnClxOSsUfBu9pkyNqjelskOEYZn18HpAxcxEZzBJoi4P7mONZzRhtUcfZNmo6bX4/Z35HJI9yYMLlohEas7VJA+O4480EGTIuOEPTX7+aARcaOGONYb0MVuo2TgM4y86Zo5bbWZcPEnsJQUoDwkVS8uO0UDSNOzPQ3nHocR4avAyr2YS1vC52UoZ93INdHEiX5X3YQHjiG5wPjn0U+ymwcT74Gir1rN9OVu9SVG2qA/UmIuvHqgLvv8LxT7KINY683KFBTLS77J6qXzTgfE/XQA2xuOy+0ehLoh0x3DX5DueRFffqu2qdV1R3cP4IGeWHoLiK/LPeQdlXH5v68THudfDn6o+dIqd0bxjDYUuPRrh2+x92IG9jf6LOzGqK+BYywj0HMyRRUQdO1gEd7zmRD/c4R6+YHPrQv3Oj63YVlexkj1YbOhlZ/q2VRed5uZeXfDTKnQjJNLkPZXeXjch7rf6kWq9Bq8pyY3WRs7KabXwQ53J5phwzazpe1Xa5hvqrv9RA52OJ+TbsqAQpfdSc57RSrv7bYT+0UIB1hpw83F2z3MY6rI65R9ajEbfeUZu1XXVHSYeOuEZbhQwLKgut6rqD9zxrHHUa3qrqeoSf6WKvh54uacGjFdtoq9heJiLTwy4dvILUidpbvAlreaTecDiEptlC866swJ1UAfve2F33Ce06jvBia1h5hf1a7TWsGXWxL0NC2zXuXBbEui3pU5VRb5T0/lZv4K0d6S9z3bqH+03920AdxIQnryz26gmk21rpvLqhieuXXWRlffga9bBjz4Lq3ht6KWqfDSs47DXT8svdWTM4XjX7n614GXKu6p6562N2OmWNUi897OBboq6t5AwPA14eOkorQF9Avmg43cySYFluqdrKgTx4UHmW+uofd8i0kQnYbgVdMtp+3CsPnbyX7vBKjatXsiAlu7dej6eD0I0GS/ZDHzVaJxq4xwJPZJkMJy9+6dDsxERgDPvtn4XlH0Iu29K7sTUGd68HxNCZa2V7eiWlgJVRbxK0m2wtdxQ1njfecG/R2ZumF5STfK8iccH+f6/PfeUNrxHw8RAqkT/z92hUCNzB5pp1DNXjkJyFpUobQYXF3WaSSiUmHf76iEvFMYcKj2uOOM7mUHnM4UzXvpZUIfP45UqE+9Iznwjjs+Bi1oDTfQ/LdICr8jEDMUN7HVLgImVgzsCE4UEDCTcauEafBqZU1y8PQQbjCsnXljNN9cw5oTsDmWWkw38W9SShJtc17GnU41PuF4hZaGpcuHI0Q0Izef6Oy/ebhPQhUsA4yd/Soe9zgoUkE04LTGPmXHgakDZzXqrQRdSSwzieMnTyOOPD21IbnmfcytWp4sc9EPN8EWq0Ngf4PGlg0mTxXki74PCg6XeAv7UsGFjE3UWsl5aVOg/uyXAtnTFHLbclw0fEfHkJg4OnRLZL0FYImlqGtL9I54Hly4AUfQU+Z5Kep+8C6xp9D5r8Bf0GWt2h39LX6XdUpG/Kf3VcpX/RQ9r8Hzk/d+sAAAAAAAABAAAAANXtRbgAAAAAzrLLtgAAAADY6RJg") format("woff");
```

Comment:
This is super grody, but the build system doesn't support file loading (and these are .scss files which adds complications as well), so the simplest way I could swap out the fonts was to inline base64 encode them. Sorry!